ajax/libs/downloadjs/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/rndme/download.js"
+    "url": "https://github.com/rndme/download"
```

Comment:
@NYCJacob You have to do two things:
1. change `downloadjs/package.json` form `"url": "https://github.com/rndme/download"` to `"url": "https://github.com/rndme/download"`
2. Add an autoupdate section as the folling to `skel-layers/package.json`.
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/jashkenas/underscore.git",
    "fileMap": [
      {
        "basePath": "src",
        "files": [
          "skel*"
        ]
      },
      {
        "basePath": "dist",
        "files": [
          "skel*"
        ]
      }
    ]
  }
```
Thank you for your contribution.